libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmTNADisclaimer.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Alert, AlertVariant, List, ListItem, Stack, StackItem } from '@patternfly/react-core';
+import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
+
+const OcmTNADisclaimer = () => {
+  const { t } = useTranslation();
+  const generalSNOFacts = (
+    <>
+      <ListItem>
+        {t('ai:Two nodes control plane OpenShift must include an additional arbiter node.')}
+      </ListItem>
+    </>
+  );
+
+  return (
+    <Alert
+      variant={AlertVariant.info}
+      title={t('ai:Requirements for Two Node control plane OpenShift')}
+      isInline
+    >
+      <Stack hasGutter>
+        <StackItem>
+          <List>{generalSNOFacts}</List>
+        </StackItem>
+      </Stack>
+    </Alert>
+  );
```

Comment:
Why isn't the `generalSNOFacts` directly in return JSX ? Also it seems that usage of `Stack` is redundant
```suggestion

  return (
    <Alert
      variant={AlertVariant.info}
      title={t('ai:Requirements for Two Node control plane OpenShift')}
      isInline
    >
      <List>
        <ListItem>
          {t('ai:Two nodes control plane OpenShift must include an additional arbiter node.')}
        </ListItem>
      </List>
    </Alert>
  );
```